client/app/achievements/achievements.html

```diff
@@ -11,34 +11,11 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```

Comment:
I think we might not want statistics to be the first thing a mobile user sees - I like seeing the mission statement and such. It describes what we do.